### ISSUE TITLE
shakeit motors unrecognised after driver update

### ISSUE THREAD

**Issue by Falconitage** [SHWotever/SimHub#1312]
**Describe the bug**
the last driver for CH340SER USB serial is wch.cn 3.8.2023.2. after a restart simhub can't recognised the arduino, I have to revert to driver 3.7.
![simhub](https://github.com/SHWotever/SimHub/assets/134137866/bf5abce9-3189-40ab-ac9b-17d457c5bcf7)

strange thing I can communicate with the arduino IDE !

![arduino ide](https://github.com/SHWotever/SimHub/assets/134137866/89523c36-89e5-4fc7-8616-5e9c1e39f187)

**Simhub version**
3.11  (Windows 11)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Log files**
Join log files available in C:\Program Files (x86)\SimHub\Logs

**Additional context**

**Comment by SHWotever** [comment]
Hi ! There is is a driver bug only affecting fake/counterfeit ch340g chips. I already updated the documentation about this issue : https://github.com/SHWotever/SimHub/wiki/Arduino--Setup#recommended-arduinos unfortunately since it's a matter of fake chip and driver compatibility there is nothing I can do on my end .

**Comment by Falconitage** [comment]
thank you, I go back to the 3.7 !

**Comment by SHWotever** [comment]
Yes, this driver mess up with counterfeit usb to serial chip (ch340g) on the Arduino.

**Comment by SHWotever** [comment]
https://www.reddit.com/r/arduino/comments/139vfux/is_ch340g_a_new_ftdigate_latest_windows_drivers/?utm_source=share&utm_medium=mweb
